backend/src/main/java/gov/cdc/usds/simplereport/db/model/ApiUser.java

```diff
@@ -1,53 +1,75 @@
 package gov.cdc.usds.simplereport.db.model;
 
 import java.util.Date;
+import java.util.UUID;
 
 import javax.persistence.Column;
+import javax.persistence.Embedded;
 import javax.persistence.Entity;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToOne;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
 
+import org.hibernate.annotations.CreationTimestamp;
 import org.hibernate.annotations.DynamicUpdate;
 import org.hibernate.annotations.NaturalId;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
 
 /**
  * The bare minimum required to link an authenticated identity to actions and data
  * elsewhere in the schema.
+ * 
+ * Since we can't use JPA auditing here, we use Hibernate {@link CreationTimestamp} and
+ * {@link UpdateTimestamp}, and feel kind of weird about it.
```

Comment:
Interested in this comment, but not worth blocking a PR.

Reply:
It is possible there's a way to do this safely and I didn't find it last night: basically, the JPA audit assumes that you can get all the audit information up front and then apply it as needed. The `ApiUser` object is part of the audit info for JPA; if you are in the process of creating an `ApiUser` and then JPA says "oh hey, I need to get the current `ApiUser` in order to finish this, and then it starts to create an `ApiUser`... you get a stack overflow.